src/services/upload/fileService.ts

```diff
@@ -121,7 +128,10 @@ export async function encryptFile(
     Get the original file name for edited file to associate it to original file's metadataJSON file 
     as edited file doesn't have their own metadata file
 */
-function getFileOriginalName(file: File) {
+function getFileOriginalName(file: File | ElectronFile) {
+    if (!(file instanceof File)) {
```

Comment:
we don't need to do this na, we are just accessing the name property which exists on both `File` and `ElectronFile`?